.github/workflows/rust.yaml

```diff
@@ -0,0 +1,178 @@
+name: Build Rust tests
```

Comment:
Why do we need a separate workflow for building and running Rust applications? It would be better to integrate this into the existing twister workflow.

Reply:
This is a mistake, I will remove this file entirely.